src/python/phenix_apps/apps/scorch/kafka/kafka.py

```diff
@@ -0,0 +1,167 @@
+import json, itertools, threading, time, sys, csv, re, os
```

Comment:
per python style standards, one import per line (ignore other code in this codebase, it needs to be cleaned up lol)